packages/workbox-cli/src/lib/errors.js

```diff
@@ -24,4 +24,14 @@ module.exports = {
   'config-validation-failed': `Your configuration is invalid:`,
   'workbox-build-runtime-error': `Service worker generation failed:`,
   'unknown-command': `Unknown command:`,
+  'no-file-extensions-found': ol`No files could be found that are suitable for
+    caching.`,
+  'no-file-extensions-selected': `Please select at least one file extension.`,
+  'sw-src-does-not-exist': ol`The source service worker file you provided does
+    not exist. Please provide the path to an existing file.`,
+  'invalid-sw-dest': ol`Please enter a valid path to use for the service worker
+    file that's created.`,
+  'glob-directory-invalid': ol`The path you entered isn't a valid directory.`,
```

Comment:
By invalid - do you mean doesn't exist or is a file or ...?

Reply:
Either. If it's not a valid directory for any reason—either it doesn't exist at all, or it exists but it's a file, you'll see that.